OPHD/Things/Structures/AirShaft.h

```diff
@@ -6,7 +6,10 @@
 class AirShaft: public Structure
 {
 public:
-	AirShaft() : Structure(constants::AIR_SHAFT, "structures/air_shaft.sprite", constants::STRUCTURE_STATE_OPERATIONAL, StructureClass::Tube)
+	AirShaft() : Structure(constants::AIR_SHAFT, "structures/air_shaft.sprite",
+		constants::STRUCTURE_STATE_OPERATIONAL,
+		StructureClass::Tube,
+		StructureID::SID_AIR_SHAFT)
```

Comment:
Minor suggestion for lining up parameters:
When any parameters are spilled to a new line, spill all parameters (including the first):
```cpp
	AirShaft() : Structure(
		constants::AIR_SHAFT,
		"structures/air_shaft.sprite",
		constants::STRUCTURE_STATE_OPERATIONAL,
		StructureClass::Tube,
		StructureID::SID_AIR_SHAFT)
```

----

Nice structuring of the commits in this PR btw. These changes were really easy to read through.